helm-system-packages-dnf.el

```diff
@@ -27,9 +27,16 @@
 ;;; Code:
 (require 'helm-system-packages)
 
+(require 'seq)
+
 (defcustom helm-system-packages-dnf-actions
-  '(("Show package(s)" . helm-system-packages-dnf-info))
-  "Actions for Helm guix."
+  (list
+   (cons "Show package(s)" #'helm-system-packages-dnf-info)
+   (cons "Install"  #'helm-system-packages-dnf-install)
+   (cons "Uninstall" #'helm-system-packages-dnf-uninstall)
+   (cons "Browse homepage URL" #'helm-system-packages-dnf-browse-url)
+   (cons "Find files" #'helm-system-packages-dnf-find-files))
+  "Actions for Helm Dnf."
```

Comment:
From the [homepage](https://rpm-software-management.github.io/): it's `DNF`, not `Dnf`.

Reply:
I fixed all of them except the parameter to `:name` in the call to `helm-system-packages-manager-create`. I can change this one as well if you want.

Reply:
I'm realizing that I've also written `guix` lowercase, but I think it's a mistake.
I don't think the name matters in the logic of the code.  I need to double-check though.